app/models/comment.rb

```diff
@@ -1,4 +1,12 @@
 class Comment < ApplicationRecord
   belongs_to :author, foreign_key: :author_id, class_name: 'User'
   belongs_to :post, foreign_key: :post_id, class_name: 'Post'
```

Comment:
- [ ] You are not supposed to specify again the class name for the Post on the comment table as in the db we have a table `Posts`, we always use class_name: attribute to reference to a table that does not exist with its name in the db, like author table that not exist, we now tell to the model to consider the user table as author. please remove that from there dear